src/Installers/Windows/AspNetCoreModule-Setup/ANCMV2/aspnetcoremodulev2.wxs

```diff
@@ -215,6 +220,50 @@
                 </Directory>
             </Directory>
             <?endif ?>
+            
+           <!-- ARM64 Support -->
+           <?if $(var.Platform) = "arm64" ?>
+           <Component Id="C_DiscoverabilityKeyARM64" Guid="2eeb90e8-28d0-4543-9c2f-843b03bd6d05" Win64="no">
+               <RegistryKey Root="HKLM" Key="$(var.DiscoverabilityKeyRoot)">
+                    <RegistryKey Key="$(var.ProductShortName)">
+                        <RegistryValue Type="integer" Name="Install" Value="1" />
+                        <RegistryValue Type="string" Name="Version" Value="$(var.ANCMMsiVersion)" />
+                    </RegistryKey>
+                </RegistryKey>
+            </Component>
+
+            <Directory Id="$(var.ProgramFilesFolder32)">
```

Comment:
Is this the correct `Id`❔ Looks like there are two `<Directory />` elements w/ the same `Id` now

Reply:
idk, i'll leave this open for now, the goal isn't to make the installers work yet, this PR is just to get something building for arm64 as the first click stop.

Reply:
My preference would be to commit a REVIEW with your question but I know how you feel about those :)